server/service/testing_utils.go

```diff
@@ -523,3 +523,28 @@ func mockSuccessfulPush(pushes []*mdm.Push) (map[string]*push.Response, error) {
 	}
 	return res, nil
 }
+
+func mdmAppleConfigurationRequiredEndpoints() [][2]string {
```

Comment:
Should probably add a comment in `handler.go` to mention that if endpoints are modified/added/removed, this list should be updated. At some point maybe we could look for a way to automate this so that the list is built out of the http handler mux.